src/javaRosa.js

```diff
@@ -951,20 +948,18 @@ define([
                 widget.setValue(defaultValue);
                 widget.handleChange();
             } else {
-                var itextItem = widget.getItextItem();
+                var value = widget.getItextValue();
 
-                if (!itextItem) {
-                    widget.setValue("");
-                    return;
+                if (!_.isString(value)) {
```

Comment:
Interesting. I have the opposite reaction to `isString` vs `isUndefined` since the latter is more specific and will not discard the value if, for example, it happens to be an integer. Not sure how that would happen, and it would probably be a bug anyway, so not a big deal.

However, this does seem a little fragile since it is on the critical path for setting the default value of the widget, and that depends on the Itext item's default value being `undefined`. Seems like we may want to do this default itext value initialization somewhere else. Maybe in `ItextModel.updateForNewMug`? ... hmm, looks like it's already being done there. Is this code even necessary?